data/Harvey_2017/metadata.yml

```diff
@@ -0,0 +1,172 @@
+    area of kwongan and woodland (previously burnt in 1960) was of moderate intensity,
+    hot in patches, and resulted in near-complete burning of the target area. The
+    1990 fire, in an area of 108 ha (previously burnt in 1965), was of moderate to
+    high intensity. It burnt through most of the kwongan at a rate of 1100mh-1, through
+    Allocasuarina woodlands in a continuous front with flame height reaching 2m and
+    at a rate of 200mh-1. In Eucalyptus woodland, the fire slowed down to 100mh-1
+    because of lower levels of ground cover and litter and reduced exposure to wind.
+    The canopy of the Eucalyptus woodland and some of the Allocasuarina woodland remained
+    unburnt; there were occasional unburnt mature Allocasuarina plants in areas where
+    the fire had been of low intensity, but the shrub and herb layers were mostly
+    or completely burnt. ^p Sampling strategy ^p To understand post-fire vegetation
+    recovery and dynamics, 21 100-m2 circular plots (14 in woodland and seven in kwongan)
+    were established in the spring of 1983, before the 1985 autumn fire, and resampled
+    in the spring of 1985, 1986, 1987, 1988, 1990, 1992, 1995, 2000 and 2015. Similarly,
+    45 100-m2 circular plots (29 in woodland and 16 in kwongan) established in the
+    spring of 1988, before the 1990 autumn fire, were resampled in the spring of 1990,
+    1991, 1992, 1994, 1997 and 2000. Within each plot, the fire-response (resprouter
+    versus non-resprouter) of each species was determined together with the reproductive
+    status of post-fire recruits for non-resprouters and post-fire resprouts for resprouters.
+  original_file: Harvey_AusTraits_working.xls in raw data folder
+  notes: none
+locations:
+  Tutanning Nature Reserve, Kwongan:
+    latitude (deg): -32.5478
+    longitude (deg): 117.3348
+    locality: Tutanning Nature Reserve, Western Australia
+    description: Kwongan
+    fire intensity: moderate to high intensity
+    precipitation, MAP (mm): 445.0
+    temperature, mean summer max (C): 31.1
+  Tutanning Nature Reserve, Kwongan and wandoo and she-oak woodland:
+    latitude (deg): -32.5478
+    longitude (deg): 117.3348
+    locality: Tutanning Nature Reserve, Western Australia
+    description: Kwongan and wandoo and she-oak woodland
+    fire intensity: moderate to high intensity
+    precipitation, MAP (mm): 445.0
+    temperature, mean summer max (C): 31.1
+  Tutanning Nature Reserve, wandoo and she-oak woodland:
+    latitude (deg): -32.5478
+    longitude (deg): 117.3348
+    locality: Tutanning Nature Reserve, Western Australia
+    description: wandoo and she-oak woodland
+    fire intensity: moderate to high intensity
+    precipitation, MAP (mm): 445.0
+    temperature, mean summer max (C): 31.1
+contexts: .na
+traits:
+- var_in: Time_from_fire_to_fGrounder
+  unit_in: mo
+  trait_name: fire_time_from_fire_to_flowering
+  entity_type: species
+  value_type: mode
+  basis_of_value: expert_score
```

Comment:
Should this be measurement maybe?

Reply:
I've changed it here. I think the other traits would be `expert_score` since they are the scoring of a specific outcome, not measurements.